db/migrations/20220712131700_nicknames.up.sql

```diff
@@ -0,0 +1,9 @@
+alter table users ADD COLUMN nickname character varying NOT NULL;
```

Comment:
When migrating an old database this yields to an error like `"Error migrating database: ERROR: column \"nickname\" of relation \"users\" contains null values (SQLSTATE=23502)"}`

Reply:
I guess you need to drop `NOT NULL`?

Reply:
Yes, but since `nickname` will be the [lightning address username](https://github.com/andrerfneves/lightning-address/blob/master/README.md) those old users without `nickname` can only be paid by its `login` (since there is now a function  that search users by login OR nickname) which is no big deal I guess ( given that logins are pretty ugly strings :) )